### PR TITLE
Add narrative documentation on botogram i18n

### PR DESCRIPTION
**Pull request by ALCC01**
Documents botogram's i18n facilities and adds narrative documentation on translating/updating/overriding default messages.

Contributes towards the completion of #46 and #33.

### CODE REVIEW

File: docs/i18n.rst [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+.. _i18n-available-translations:
+
+Available translations
+======================
+
+The following languages are currently included in the botogram package:
+
+   * ``en`` (English)
+   * ``it`` (Italian)
```
Comment (pietroalbini) [inline docs/i18n.rst]: Remove the spaces before the bullets, they create an indented block.

File: docs/i18n.rst [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+.. _i18n-update-translation:
+
+Updating a translation
+======================
+
+As botogram evolves, more message will probably be added to the codebase, and it
+is also possible for currently included translations to contain mistakes.
+
+The workflow for updating a translation is basically the same as the one
+described in `i18n-new-language`_, but you may also need to use ::
```
Comment (pietroalbini) [inline docs/i18n.rst]: A label is missing for this link, and apparently it's not captured by Sphinx.

File: docs/i18n.rst [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+should check your diff and avoid committing any change that doesn't impact the
+actual translation and the source code references in comments.
+
+While trivial, it would be nice if you also changed the ``PO-Revision-Date``
+header to reflect your changes.
+
+.. _i18n-overriding:
+
+Overriding default messages
+===========================
```
Comment (pietroalbini) [inline docs/i18n.rst]: Maybe this section should be before the translating ones, I feel it's more important.

**Comment by ALCC01** [comment]
d16bb01 should do it.

**Pull request merged**